pino.js

```diff
@@ -208,13 +208,15 @@ Object.defineProperty(pinoPrototype, 'child', {
 })
 
 function pinoWrite (obj, msg, num) {
-  var s = this.asJson(obj, msg, num)
+  var t = this.time()
+  var s = this.asJson(obj, msg, num, t)
   var stream = this.stream
   if (this.cache === null) {
     if (stream[needsMetadata]) {
       stream.lastLevel = num
       stream.lastMsg = msg
       stream.lastObj = obj
+      stream.lastTime = t.slice(8)
```

Comment:
is there a chance that `t` won't be a string (does `this.time` originate externally?)

Reply:
No, we are assuming it's a string everywhere anyway.

Reply:
well then, LGTM